rfcs/0098-community-team.md

```diff
@@ -0,0 +1,190 @@
+---
+feature: community-team
+start-date: 2021-07-31
+author: Irenes (Irene Knapp)
+co-authors: ashkitten
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: (will contain links to implementation PRs)
+---
+
+# Summary
+[summary]: #summary
+
+Establish an official community team to make moderation decisions, model social norms, and mediate interpersonal problems.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not sustainable to have to call on Graham any time there's a spammer or conflict that requires moderation, and we'd like to help the community become more self-regulating.
+
+# Goals
+[goals]: #goals
+
+* Model and enforce social norms
+* Mediate disputes when doing so is beneficial to the community
+* Respect the privacy of community members
+* Foster newcomers and develop their leadership potential
+* Moderate all places where community members interact, including Discourse, Matrix, and GitHub
+* Ensure that collaboration on shared technical goals is possible
```

Comment:
The goals aren't ranked. Everyone will have different opinions on which is the most important, and we should discuss that in this thread to the extent that they may conflict. The purpose of the "goals" section is just to provide a rubric for the discussion we're doing now, to evaluate how well the rest of the RFC is meeting those goals.

That said, it might indeed be helpful to sort the goals in some way that makes intuitive sense to people. We tried our best to do that but we're not perfect. If you would like to take a pass at coming up with a different sorting for the goals list, feel free. Just keep in mind while doing it that the order is purely for ease of reading, to convey the ideas as clearly as possible, and doesn't imply any prioritization.

Reply:
It would be too much work to rank all the items. Each individual has different priorities in life and would rank them differently. That's why I think it's important that the team's focus be on making collaboration around NixOS possible, not become enforcers of morality.